s2wasm_known_gcc_test_failures.txt

```diff
@@ -3,3 +3,9 @@
 
 # too few parameters to function in call. got 8, expected 9
 pr44942.c.s.wast
+
+# New in 0xc:
+# call_indirect with no table
```

Comment:
Oh interesting I thought I fixed that. maybe i never checked the fix in. i'll take a look.